prism/src/main/java/org/apache/falcon/service/BacklogMetricEmitterService.java

```diff
@@ -100,6 +100,41 @@ protected SimpleDateFormat initialValue() {
     private static ConcurrentHashMap<Entity, List<MetricInfo>> entityBacklogs = new ConcurrentHashMap<>();
 
     @Override
+    public void onAdd(Entity entity) throws FalconException{
+        //DO Nothing
```

Comment:
On addition of an entity whose start time is back dated, should not backlogemitterservice populate the SLAs for those instances between start time and current time?